docs/contributor/resources/03-moduletemplate.md

```diff
@@ -75,6 +75,32 @@ spec:
 - documentation: The link to the documentation of the module.
 - icons: A list of icons of the module, each with a name and link.
 
+### **.spec.manager**
+
+The `manager` field provides information for identifying a module's resource that can be used as indicator for the installation readiness of the module. Typically, this is the manager Deployment of the module. In exceptional cases, it may also be another resource. If the resource is not namespaced, such as the module CR's `CRD`, the namespace can be optional.
+
+For example, if the module has a manager `Deployment` in the `kyma-system` namespace:
+
+```yaml
+spec:
+  manager:
+    group: apps
+    version: v1
+    kind: Deployment
+    namespace: kyma-system
+    name: [module manager name]
+```
+
+For a module without a manager, such as [cloud manager](https://github.com/kyma-project/cloud-manager):
+
+```yaml
+spec:
+  manager:
+    group: apiextensions.k8s.io
+    version: v1
+    kind: CustomResourceDefinition
+    name: [module CRD name]
+```
```

Comment:
```suggestion
```

Thinking about it, maybe we should remove this example entirely. Just to not give the impression that this is something that should be done.

Reply:
if we need to have this use case, I think we should mention it, for documentation purpose, it also brings value,  implicitely as hidden rule which applied for certain modules could lead to more confusion.

However, as for whether it make sense of making CRD as the manager, we can open a dicussion on this. It might be better to leave this field optional, assuming that for modules without a manager, they are considered ready by default. Until a decision is made, I believe we should keep this use case as a valid example